### ISSUE TITLE
Cropping slide source

### ISSUE THREAD

**Issue by saerdnaer** [voc/voctomix#60]
In some cases, e.g. 4:3 slides on 16:9 projector, it would be nice to have a flag to set a source to 4:3, so there are no black bars in side by side views.

**Comment by MaZderMind** [comment]
I think this is a duplicate of #31 - @saerdnaer would you agree?

**Comment by saerdnaer** [comment]
Well, so far the case described there don't cover this case:
```
Which option describes the Problem with the Slides best?
a) The Slides are compressed
b) The Slides are stretched
c) The Slides are cropped```

So probably add: ```d) The Slides are 4:3```. But I also don't understand how to fix c)... You can't add information which isn't there, in software.

**Comment by MaZderMind** [comment]
closed as duplicate of #31
